boards/arm/nucleo_f412zg/nucleo_f412zg_defconfig

```diff
@@ -11,15 +11,19 @@ CONFIG_HW_STACK_PROTECTION=y
 
 CONFIG_SERIAL=y
 
-# console
+# Console
 CONFIG_CONSOLE=y
 CONFIG_UART_CONSOLE=y
 
-# enable GPIO
+# Enable GPIO
 CONFIG_GPIO=y
 
 # Enable Clocks
 CONFIG_CLOCK_CONTROL=y
 
-# enable pin controller
+# Enable pin controller
 CONFIG_PINCTRL=y
+
+# Enable CAN
+CONFIG_CAN=y
```

Comment:
this is mixing style changes which are not related. CAN is not a minimal peripheral, it should not be in board defconfig.

Reply:
Thank you for your review . I have implemented the changes you requested.